### PR TITLE
Feat: Module gcp/wildcard-sslcert

### PR DESCRIPTION
**Pull request by spartan-tuantran**
### Why
Add new module gcp/wildcard-sslcert

### What
Add new module gcp/wildcard-sslcert

### Solution
Add new module gcp/wildcard-sslcert

## Types of Changes

- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] 🚀 New feature (non-breaking change which adds functionality)
- [ ] 🕷 Bug fix (non-breaking change which fixes an issue)
- [ ] 👏 Performance optimization (non-breaking change which addresses a performance issue)
- [ ] 🛠 Refactor (non-breaking change which does not change existing behavior or add new functionality)
- [ ] 📗 Library update (non-breaking change that will update one or more libraries to newer versions)
- [ ] 📝 Documentation (non-breaking change that doesn't change code behavior, can skip testing)
- [ ] ✅ Test (non-breaking change related to testing)
- [ ] 🔒 Security awareness (changes that effect permission scope, security scenarios)

### CODE REVIEW

File: gcp/wildcard-sslcert/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Terraform Wildcard SSLCert Module
+
+This Terraform module creates a cert map for a wildcard domain.
+
+## Usage
+```hcl
+module "wildcard-sslcert" {
+  source = "../../"
+  domain              = "example.com"
```
Comment (spartan-ytruong) [inline gcp/wildcard-sslcert/README.md:9]: this section has not been formatted

File: gcp/wildcard-sslcert/examples/completes/main.tf [hunk 2]
```diff
@@ -0,0 +1,4 @@
+module "wildcard-sslcert" {
+  source = "../../"
+  domain              = "example.com"
```
Comment (spartan-ytruong) [inline gcp/wildcard-sslcert/examples/completes/main.tf:3]: you can run `terraform fmt --recursive` from root directory

File: gcp/wildcard-sslcert/examples/completes/README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# VPC
```
Comment (spartan-ytruong) [inline gcp/wildcard-sslcert/examples/completes/README.md:1]: it is not renamed

**Review by spartan-ytruong — CHANGES_REQUESTED** [review]

File: gcp/wildcard-sslcert/README.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
+# Terraform Wildcard SSLCert Module
+
+This Terraform module creates a cert map for a wildcard domain.
+
+## Usage
+```hcl
+module "wildcard-sslcert" {
+  source = "../../"
```
Comment (spartan-ytruong) [inline gcp/wildcard-sslcert/README.md:8]: source  = "github.com/spartan-stratos/terraform-modules//gcp/wildcard-sslcert?ref=v0.1.4"

**Review by spartan-toannhan — APPROVED** [review]

**Pull request merged**